### ISSUE TITLE
From the results，seem not good!

### ISSUE THREAD

**Issue by BoxFishLab** [zhengchen1999/RGT#4]

**Comment by zhengchen1999** [comment]
Maybe you can provide relevant examples. Or provide other methods that are superior to ours.

**Comment by zelenooki87** [comment]
Yeah, photos are not restored. Seems like only bicubic upscale is done. From test single datased x4. (example)
![test_2_x4](https://github.com/zhengchen1999/RGT/assets/75204635/daf68ab5-fe1c-4369-86d9-80ded6a05618)

**Comment by zhengchen1999** [comment]
It seems there's an issue. However, the pre-trained models we provide weren't trained under blind SR conditions. @Phhofm has offered some variants that might solve this problem in #3.

**Comment by Phhofm** [comment]
Yeah it makes sense for it to look this way, official pretrains are not meant to handle degradations of this level.
My variants so far would also not work too well on this specific input, 4xRealWebPhoto is meant for photographs for example.
What would work best on this specific input were a model trained on nomos_uni with otf.
I can demonstrate what I mean, the following is an output of an RGT-S model I just trained for only 8000 iterations to demonstrate what I mean, a model fully trained this way would be able to handle this specific input best:
![test_2_8000](https://github.com/zhengchen1999/RGT/assets/14755670/9196d9e2-7808-4daf-bd98-c536a1cdb641)

**Comment by SuSung-boy** [comment]
![2024_cro11111p](https://github.com/zhengchen1999/RGT/assets/17896245/70f3c077-133f-4a5c-8d33-5ff953820660)
do you have any idea that I get result img like this?
how my steps got wrong?
